### PR TITLE
Rename get user consumer from GetUserInfo to GetUserData

### PR DESCRIPTION
**Pull request by MariyaSin**
Renamed Consumer, requestModel, responseModel from GetUserInfo to GetUserData

PR is a sub-task №231 of task  http://outsourcing.nat.tepkom.ru:20080/redmine/issues/165

### CODE REVIEW

**Review by borovHokage — APPROVED** [review]

File: test/UserService.Broker.UnitTests/Consumers/GetUserInfoConsumerTests.cs [hunk 1]
```diff
@@ -25,7 +25,7 @@ internal class OperationResult<T> : IOperationResult<T>
         public T Body { get; set; }
     }
 
-    internal class UserInfoResponse : IGetUserResponse
+    internal class UserInfoResponse : IGetUserDataResponse
```
Comment (DarkFelas) [inline test/UserService.Broker.UnitTests/Consumers/GetUserInfoConsumerTests.cs:28]: UserInfoResponse ????????

**Comment by DarkFelas** [comment]
Do not see task link in description

**Comment by DarkFelas** [comment]
I can not merge this one because u have unresolved conversation

File: test/UserService.Broker.UnitTests/Consumers/GetUserInfoConsumerTests.cs (hunk 1 above)
Comment (MariyaSin) [reply test/UserService.Broker.UnitTests/Consumers/GetUserInfoConsumerTests.cs]: fixed

File: test/UserService.Broker.UnitTests/Consumers/GetUserDataConsumerTests.cs [hunk 2]
```diff
@@ -105,8 +105,8 @@ public async Task ShouldResponseUserInfoResponse()
                 Assert.True(response.Message.IsSuccess);
                 Assert.AreEqual(null, response.Message.Errors);
                 SerializerAssert.AreEqual(expected, response.Message);
-                Assert.That(consumerTestHarness.Consumed.Select<IGetUserRequest>().Any(), Is.True);
-                Assert.That(harness.Sent.Select<IOperationResult<IGetUserResponse>>().Any(), Is.True);
+                Assert.That(consumerTestHarness.Consumed.Select<IGetUserDataRequest>().Any(), Is.True);
+                Assert.That(harness.Sent.Select<IOperationResult<IGetUserDataResponse>>().Any(), Is.True);
```
Comment (DarkFelas) [inline test/UserService.Broker.UnitTests/Consumers/GetUserDataConsumerTests.cs:109]: Use Assert.True()

File: test/UserService.Broker.UnitTests/Consumers/GetUserDataConsumerTests.cs [hunk 3]
```diff
@@ -105,8 +105,8 @@ public async Task ShouldResponseUserInfoResponse()
                 Assert.True(response.Message.IsSuccess);
                 Assert.AreEqual(null, response.Message.Errors);
                 SerializerAssert.AreEqual(expected, response.Message);
-                Assert.That(consumerTestHarness.Consumed.Select<IGetUserRequest>().Any(), Is.True);
-                Assert.That(harness.Sent.Select<IOperationResult<IGetUserResponse>>().Any(), Is.True);
+                Assert.That(consumerTestHarness.Consumed.Select<IGetUserDataRequest>().Any(), Is.True);
```
[the remark this answers is not in the document]
Comment (MariyaSin) [reply test/UserService.Broker.UnitTests/Consumers/GetUserDataConsumerTests.cs]: Done

File: test/UserService.Broker.UnitTests/Consumers/GetUserDataConsumerTests.cs (hunk 2 above)
Comment (MariyaSin) [reply test/UserService.Broker.UnitTests/Consumers/GetUserDataConsumerTests.cs]: Done

**Review by TIove — APPROVED** [review]

**Review by borovHokage — APPROVED** [review]

**Pull request merged**